systemd/schema-registry.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Schema Registry
+Documentation=http://docs.confluent.io/current/schema-registry/docs/index.html
+Requires=network.target
+Wants=kafka-server.service
+After=network.target kafka-server.service
+
+[Service]
+Type=forking
```

Comment:
yes it will be running as root by default.  In the future this can be enhanced to run as a specific user.